src/renderer/components/+extensions/extensions.tsx

```diff
@@ -232,10 +240,10 @@ export class Extensions extends React.Component {
     // copy files to temp
     await fse.ensureDir(this.getExtensionPackageTemp());
 
-    requests.forEach(req => {
-      const tempFile = this.getExtensionPackageTemp(req.fileName);
-      fse.writeFileSync(tempFile, req.data);
-    });
+    for (const request of requests) {
```

Comment:
Replacement to `for-of` because `fse.writeFile` should break whole method & throw errors if any?

Reply:
Yeah, if I used forEach with async callback then we could lose errors